api/routes/api_games.py

```diff
@@ -16,17 +16,12 @@ def handle_exception(e):
 @games_app.route('/', methods=['GET'])
 @cache.cached(60*1.3)
 def get_stream_info_today_endpoint():
-   try:
-      games = get_stream_info_today(session)
-      return games
-   except Exception as e:
-       print(e)
+    games = get_stream_info_today(session)
+    return games
```

Comment:
why did we remove try-except here?

Reply:
because I moved all the logic under the hood into functions
I thought it would be better that way